### PR TITLE
Add acquia/verify endpoint

### PR DESCRIPTION
**Pull request by malachy-mcconnell**

### CODE REVIEW

**Comment by malachy-mcconnell** [comment]
I will squash the commits after I make the code-sniff corrections

File: modules/diagnostic/src/VerifyMapping.php [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+    // Description (site description in locale?)
+    $configSystemSite = $this->configFactory->get('system.site');
+
+    // Translations? Is the config locale aware?
+    $description = $configSystemSite->get('name');
+    $description .= " (" . $configSystemSite->get('slogan') . ").";
+    $response['description'] = $description;
+
+    // TODO (malachy): Add in the loop back to test connection
```
Comment (csibar) [inline modules/diagnostic/src/VerifyMapping.php:172]: Same question as in commerce-manager-magento counterpart: is this something you want to implement in this PR? Just want to make sure it doesn't miss any functionality. :)

Comment (malachy-mcconnell) [reply modules/diagnostic/src/VerifyMapping.php:172]: There will be a follow-on story to implement it.

Comment (csibar) [reply modules/diagnostic/src/VerifyMapping.php:172]: Yeah, heard it on the demo, sorry.

**Review by PoissonJ — APPROVED** [review]
Nice 👍

**Pull request merged**